priv/tailwind-plus-radiant/radiant-js/src/components/map.jsx

```
@@ -0,0 +1,48 @@
+        />
+      </svg>
+      <img
+        alt=""
+        src={src}
+        className="absolute top-[4px] left-[7px] size-6 rounded-full"
+      />
+    </motion.div>
+  )
+}
+
+export function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)